docs/README.md

```diff
@@ -60,15 +60,8 @@ architecture itself then is just a result of all accepted ADRs, which have not
 been deprecated or superseeded. An up-to-date index of still relevant ADRs is
 kept [here](./adr/README.md).
 
-The following diagram represents the internal structure of the Hydra Node and the interactions between its components.
-
-![](images/hydra-architecture-direct.jpg)
-
-**Legend**:
-- Grayed boxes represent components which are not developed yet
-- Black boxes represent components which are expected to be used as _black box_, eg. without any knowledge of their inner workings.
-- Arrows depict the flow of data (Requests, messages, responses...)
-- We represent some components that are not part of the Hydra node proper for legibility's sake
+Please refer to the [inline documentation](https://input-output-hk.github.io/hydra-poc/haddock/hydra-node/index.html)
+of each module for more details.
```

Comment:
A list of notable modules here https://input-output-hk.github.io/hydra-poc/haddock/hydra-node/index.html would be nice. Just mentioning as we had a list before, but we can also keep on improving on this later.

Reply:
I think the list of notable modules should just be the list of level 1 modules: The structure of modules is part of the architecture of the system hence there's no need to repeat information and potentially introduce discrepancies.